### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by 25lid**

### CODE REVIEW

File: src/main/kotlin/frc/robot/Robot.kt [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+     * chooser code works with the Java SmartDashboard. If you prefer the
+     * LabVIEW Dashboard, remove all of the chooser code and uncomment the
+     * getString line to get the auto name from the text box below the Gyro
+     *
+     *
+     * You can add additional auto modes by adding additional comparisons to
+     * the switch structure below with additional strings. If using the
+     * SendableChooser make sure to add them to the chooser code above as well.
+     */
+    override fun autonomousInit() {
```
Comment (Kanishk-Pandey) [inline src/main/kotlin/frc/robot/Robot.kt:56]: This isn't being used right now

**Review by Kanishk-Pandey — APPROVED** [review]
Looks good

**Review by okCookie — APPROVED** [review]
looks good

**Review by trevnels — APPROVED** [review]

**Review by a1cd — APPROVED** [review]
Sicky wicky lookks goood!

**Review by Srao2020 — APPROVED** [review]
Great

**Pull request merged**